python/get_regress_index_field.py

```diff
@@ -21,49 +21,49 @@
 from standardize_time_series import standardize_time_series
 from regress_index_field import regress_index_field
 from aggregate_time_series_data import aggregate_time_series_data
- 
+
 def get_regress_index_field (indir,
-			   casename,
-			   field_name,
-			   interp_grid,
-			   interp_method,
-			   begin_yr,
-			   end_yr,
-			   begin_month,
-			   end_month,
-			   aggregate,
-			   lag,
-			   reg,
-			   reg_name,
-			   no_ann,
-			   stdize,
-			   debug = False):
-
-
-	print __name__, 'casename: ', casename
-
-	index, n_months_season, units_index = get_reg_seasonal_avg (
-							  indir         = indir[1],
-							  casename      = casename[1],
-							  field_name    = field_name[1],
-							  interp_grid   = interp_grid[1],
-							  interp_method = interp_method[1],
-							  begin_yr      = begin_yr,
-							  end_yr        = end_yr,
-							  begin_month   = begin_month[1],
-							  end_month     = end_month[1],
-							  reg           = reg[1],
-							  aggregate     = aggregate,
-							  debug         = debug)
-
-
-	if aggregate == 0 and no_ann == 1:
-		index_no_ann = remove_seasonal_cycle_monthly_data(index, n_months_season, debug = debug)
-		index = index_no_ann
-
-	if stdize == 1:
-		index_stddize = standardize_time_series(index)
-		index = index_stddize
+               casename,
+               field_name,
+               interp_grid,
+               interp_method,
+               begin_yr,
+               end_yr,
+               begin_month,
+               end_month,
+               aggregate,
+               lag,
+               reg,
+               reg_name,
+               no_ann,
+               stdize,
+               debug = False):
+
+
+    print __name__, 'casename: ', casename
+
+    index, n_months_season, units_index = get_reg_seasonal_avg (
+                              indir         = indir[1],
+                              casename      = casename[1],
+                              field_name    = field_name[1],
+                              interp_grid   = interp_grid[1],
+                              interp_method = interp_method[1],
+                              begin_yr      = begin_yr,
+                              end_yr        = end_yr,
+                              begin_month   = begin_month[1],
+                              end_month     = end_month[1],
+                              reg           = reg[1],
+                              aggregate     = aggregate,
+                              debug         = debug)
+
+
+    if aggregate == 0 and no_ann == 1:
+        index_no_ann = remove_seasonal_cycle_monthly_data(index, n_months_season, debug = debug)
+        index = index_no_ann
+
+    if stdize == 1:
+        index_stddize = standardize_time_series(index)
+        index = index_stddize
 
 
         field, lat_reg, lon_reg, area_reg, units_out = read_monthly_data_ts(indir = indir[0],
```

Comment:
I believe this line is the problem.  The indentation is wrong and it ended up in the if statement above, which it should not have.